setup.cfg

```diff
@@ -119,7 +119,7 @@ exclude =
 
 [pydocstyle]
 convention = google
-match_dir = ^(?!tests).*
+match_dir = ^(?!tests|venv|build|scripts).*
```

Comment:
Why did you add this ? I don't get clearly the point

Reply:
I don't want to spend time linting those local directories only used for development and containing third-party sources.